LBS/main/controller/location.py

```diff
@@ -121,7 +129,7 @@ def get_location():
     # if len(data["gsm_cells"]) == 2:
     #     abort(400, f'Need at least 2 cell towers to determine location, got 2')
 
-    if len(data["gsm_cells"]) == 1 or 2:
+    if len(data["gsm_cells"]) in [1, 2]:
```

Comment:
The previous line returns true regardless of the length of data["gsm_cells"]  and causes a bug so i changed it to check if len is 1 or 2 by using 'in [1,2]'